pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -595,6 +590,29 @@ func TestNewCredentialFromLinkedDataProof_EcdsaSecp256k1Signature2019(t *testing
 
 //nolint:lll
 func TestNewCredential_JSONLiteralsNotSupported(t *testing.T) {
+	cmtrJSONLD := `
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "name": "http://schema.org/name",
+    "description": "http://schema.org/description",
+    "image": {
+      "@id": "http://schema.org/image",
+      "@type": "@id"
+    },
+    "hetc": "https://w3c-ccg.github.io/vc-json-schemas/example/cmtr/v0.2/cmtr-v0.2#",
+    "cmtr": {
+      "@id": "hetc:cmtr",
+      "@type": "@json"
+    }
+  }
+}
+`
+
+	docLoader := createTestJSONLDDocumentLoader()
+	addJSONLDCachedContext(docLoader, "https://www.w3.org/2018/credentials/examples/v1", cmtrJSONLD)
```

Comment:
why `https://www.w3.org/2018/credentials/examples/v1` - doesn't seem related to the context.

Reply:
It was a bug - corrected.